### PR TITLE
TE: adding API for Anomaly feedback support

### PR DESCRIPTION
**Pull request by puneetjaiswal**
- Adding API to support anomaly feedback
- Refactoring MetricAggregationFunctions related code path.

### CODE REVIEW

File: thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/client/pinot/PinotThirdEyeClient.java [hunk 1]
```diff
@@ -291,7 +292,7 @@ public static void main(String[] args) throws Exception {
     builder.setCollection("login_mobile");
     builder.setStartTimeInclusive(DateTime.parse("2016-05-11"));
     builder.setEndTimeExclusive(DateTime.parse("2016-05-17"));
-    builder.setMetricFunctions(Lists.newArrayList(new MetricFunction(MetricFunction.Function.SUM, "loginAttempt")));
+    builder.setMetricFunctions(Lists.newArrayList(new MetricFunction(MetricAggFunction.SUM, "loginAttempt")));
```
Comment (jiateoh) [inline thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/client/pinot/PinotThirdEyeClient.java]: Please go ahead and delete this main method if it serves no practical purpose.

**Comment by kishoreg** [comment]
LGTM. while you are on this, can you add two more fields -  comments/description, issue status (fixed, in progress, wont fix, recovered etc).

File: thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/client/pinot/PinotThirdEyeClient.java (hunk 1 above)
Comment (puneetjaiswal) [inline thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/client/pinot/PinotThirdEyeClient.java]: Will do.

**Comment by puneetjaiswal** [comment]
I will do it as a separate commit. It seems that having a separate table to handle feedbacks would be a better idea. We can use same table to handle feedbacks for AnomalyResult and MergedAnomalyResult both type of entities.

**Pull request merged**